src/Nancy/Json/Simple/NancySerializationStrategy.cs

```diff
@@ -0,0 +1,217 @@
+﻿namespace Nancy.Json.Simple
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using Nancy.Extensions;
+
+    internal class NancySerializationStrategy : PocoJsonSerializerStrategy

```

Comment:
Changes required to this class

- Change the class to `public`
- Public members needs XML documentation (including the class itself)
- Remove double blank lines